setup.py

```diff
@@ -44,15 +44,16 @@
             'default.tensor = pennylane.beta.plugins.default_tensor:DefaultTensor',
             'default.tensor.tf = pennylane.beta.plugins.default_tensor_tf:DefaultTensorTF',
             ],
+        'console_scripts': [
+                'pl-device-test=pennylane.plugins.tests:cli'
+            ]
         },
     'description': 'PennyLane is a Python quantum machine learning library by Xanadu Inc.',
     'long_description': open('README.rst').read(),
     'provides': ["pennylane"],
     'install_requires': requirements,
-    'command_options': {
-        'build_sphinx': {
-            'version': ('setup.py', version),
-            'release': ('setup.py', version)}}
+    'package_data': {'pennylane': ['plugins/tests/pytest.ini']},
+    'include_package_data': True
```

Comment:
If not for this addition, would `plugins/tests/pytest.ini` be ignored when the test suite is invoked by using the `cli`?

Reply:
The issue is that the `pytest.ini` file isn't included in the packaged wheel without this being included! So this forces it to be packaged in the wheel.

By default `setuptools` will only package up `.py` files.